test/integration/puppetlabs/services/jruby/jruby_pool_int_test.clj

```diff
@@ -37,6 +37,80 @@
    :ssl-key     localhost-key
    :ssl-ca-cert ca-cert})
 
+(def script-to-check-if-constant-is-defined
+  "begin; InstanceID; true; rescue NameError; false; end")
```

Comment:
n/m, totally misunderstood that the warning was coming from the constant being initialized twice from `set-constants-and-verify` rather than from checking from here if the constant is defined.